src/main/java/com/boardgamesworld/bgrental/boardgame/infrastructure/api/BoardGameController.java

```diff
@@ -0,0 +1,124 @@
+package com.boardgamesworld.bgrental.boardgame.infrastructure.api;
+
+import com.boardgamesworld.bgrental.boardgame.domain.BoardGame;
+import com.boardgamesworld.bgrental.boardgame.domain.BoardGameFacade;
+import com.boardgamesworld.bgrental.boardgame.domain.InvalidBoardGameException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.net.URI;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@RestController
+@RequestMapping("/boardgames")
+public class BoardGameController {
+
+    private BoardGameFacade boardGameFacade;
+    private Logger logger = LoggerFactory.getLogger(BoardGameController.class);
+
+    public BoardGameController(BoardGameFacade boardGameFacade) {
+        this.boardGameFacade = boardGameFacade;
+    }
+
+    @GetMapping
+    public ResponseEntity<List<BoardGameDto>> getAllBoardGames() {
```

Comment:
zwracanie samej listy to nie jest najlepszy pomysł. JSON wyglada tak 
```
[ 
 { }, { }
]
```
Dołożenie nowego pola jest niemożliwe, dlatego powszechną praktyką jest owrapowanie listy w nową klaske, np. BoardGameResponse i w rezultacie JSON bedzie wygladac tak
```
{ 
 "games": [ {}, {} ],
 "totalCount": 10, 
  // kolejne pola które możemy dodać nie łamiąc kontraktu API - jest to zmiana addytywna
}